### DISCUSSION TITLE
Add anti noclip or pvp zones

### DISCUSSION THREAD

**Discussion by ELPatitoLoco** [TAServers/issues#22] [Addon Suggestions]
![image](https://github.com/TAServers/issues/assets/130762237/27775689-6281-4ecc-aa2a-cbbf549b554f)
It would be good to add a entitie that creates a little zone in wich you can't noclip in or when you join it it automatically puts you in pvp mode, this would be good for that annoyng people that follow you around with noclip while messing with you, I would like to suggest this since going to vec(-4000,-4000,-4000) is no longer a viable solution, if its required I could do the addon myself or fork the one in the image.

**Comment by Derpius** [comment]
A pvp-only / buildmode-only sent seems like a nice small manageable project. If you'd like to pick that up go ahead, otherwise it might be worth asking around the Discord to see if anyone wants to work on it.

Zones could be coloured like in the image above, with red for pvp and blue for build

**Reply by ELPatitoLoco** [reply]
Ok, I'll try to pick it up, also I want to implement that in the map so it should help
